domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectBuilder.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.domain.classify
+
+import org.oppia.android.app.model.Fraction
+import org.oppia.android.app.model.InteractionObject
+import org.oppia.android.app.model.ListOfSetsOfHtmlStrings
+import org.oppia.android.app.model.StringList
+import javax.inject.Inject
+
+/** Helper class which can provide with the [InteractionObject] of a particular ObjectTypeCase. */
+class InteractionObjectBuilder @Inject constructor() {
```

Comment:
Lets rename this as `InteractionObjectTestBuilder`

Even this suggestion is not the best but still it will not conflict with the `InteractionObjectBuilder` which is available via proto.

Reply:
ah, I missed that proto point. Thanks